packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/pageHeight.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * Script to set the document height.
+ *
+ * @param {String} height - The height to set the document to.
```

Comment:
Is height a string to account for units? IE: `20px`

Reply:
Yeah exactly. It's used in these places:
- https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/makeAreaScreenshot.js#L56
- https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/makeAreaScreenshot.js#L98